Utest/rimea_tests/test_1/runtest_rimea_1.py

```diff
@@ -48,8 +50,11 @@ def run_rimea_test1(inifile, trajfile):
                      argv[0], must_min_time, evac_time, must_max_time)
         exit(FAILURE)
 
+
 if __name__ == "__main__":
+    start_time=time.time()
```

Comment:
Instead of repeating the same code over and over in each test file, isn't there a way to make JPSRunTestDriver emit the output you need?

Reply:
Year this would be nicer, but if we change this we also have to change the python scrips for the Juelich test and all other test witch are working witch the JPSRunTestDriver.
So i think this will be a little Project for the future but now  I just want to get Rime running.